deepspeed/utils/zero_to_fp32.py

```diff
@@ -52,7 +61,7 @@ def get_model_state_file(checkpoint_dir, zero_stage):
 
 def get_optim_files(checkpoint_dir):
     # XXX: need to test that this simple glob rule works for multi-node setup too
-    optim_files = sorted(glob.glob(os.path.join(checkpoint_dir, "*_optim_states.pt")))
+    optim_files = sorted(glob.glob(os.path.join(checkpoint_dir, "*_optim_states.pt")), key=natural_keys)
```

Comment:
Can you please share unit test for this case?

Reply:
Unfortunately, our CI does not currently support multi-node testing :(

Maybe a unit test that calls `get_optim_files` directly and passes a `tmpdir` path where some empty files were created with the `*_optim_states.pt` naming style.

Reply:
Makes sense to me. I will prepare one.

Reply:
@jeffra @tjruwase unittest is added